### DISCUSSION TITLE
Reference book of OCR Python (Open CV, Tesseract, Keras, etc)

### DISCUSSION THREAD

**Discussion by DeanAlexander27** [teamalgoritma/community#545] [Tips]
Dear Seniors, Instructors hope you all in well,
Would you mind giving me a reference books of OCR Python to my final thesis.
I've got quite interesting OCR's book by dr. Adrian Rosebrock, but the price was high.
thank you.

**Comment by finesaaa** [comment]
Dear Pak, @DeanAlexander27,
Thank you for the question; we've accepted your question.
For me, actually, I learn OCR from their official documentation, such as https://tesseract-ocr.github.io/docs/ (with learn LSTM model, Image Processing, etc that related to OCR (example: https://youtu.be/qjrad0V0uJE))
However, we'll still try to help search the other references too. Please wait for this

**Comment by idoali** [comment]
Thanks for the question

There is this famous book called "Web Scraping with Python" published by O'reilly. As the name implies, it doesn't only talk about OCR but web scraping and taking information from texts as a whole.

All the information about OCR itself can be found on its documentation, as mentioned above. But, it's better to start your journey from a simple blog post. It is easier and more enjoyable.

For a book that specifically talks about OCR, we haven't found it yet.
